### ISSUE TITLE
Improve SEO title and description

### ISSUE THREAD

**Issue by kinueng** [OpenLiberty/guide-bean-validation#26]
The attributes below need to be set if we want to improve the SEO of this guide.  

Example from the guide-getting-started guide. 
 https://raw.githubusercontent.com/OpenLiberty/guide-getting-started/master/README.adoc
```
:page-seo-title: Open Liberty tutorial
:page-seo-description: Find out how to run microservices on Open Liberty 
```

The guide is currently using our default SEO title and description.  Example from slack.
![image](https://user-images.githubusercontent.com/31117513/44666729-a623d480-a9de-11e8-8074-72add25a5e25.png)

**Comment by evelinec** [comment]
Thank you @kinueng for opening the issue. We're working on the SEO for all guides. We'll cover this one too.

**Comment by evelinec** [comment]
@proubatsis can you add this guide to our list. Thanks.

**Comment by yeekangc** [comment]
Addressed via #35 and #36.
